oci/oci.go

```diff
@@ -474,7 +474,10 @@ func (r *Runtime) StopContainer(c *Container) error {
 func (r *Runtime) DeleteContainer(c *Container) error {
 	c.opLock.Lock()
 	defer c.opLock.Unlock()
-	return utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.Path(c), "delete", c.name)
+	if _, err := utils.ExecCmd(r.Path(c), "delete", c.name); err != nil && !strings.Contains(err.Error(), "does not exist") {
```

Comment:
https://github.com/opencontainers/runc/pull/1451 has been merged, let's use `-f` so we ignore not exist error while still bubbling up other errors, I'll fix it.